classes/classes/Items/MutationsHelper.as

```diff
@@ -41,41 +44,45 @@ package classes.Items
 
 			if (changes < changeLimit && player.armType != ARM_TYPE_HUMAN) {
 				if ([ARM_TYPE_HARPY, ARM_TYPE_SPIDER, ARM_TYPE_SALAMANDER].indexOf(player.armType) >= 0)
-					outputText("\n\nYou scratch at your biceps absentmindedly, but no matter how much you scratch, it isn't getting rid of the itch.  Glancing down in irritation, you discover that");
+					message += "\n\nYou scratch at your biceps absentmindedly, but no matter how much you scratch, it isn't getting rid of the itch.";
 
 				switch (player.armType) {
 					case ARM_TYPE_HARPY:
-						outputText(" your feathery arms are shedding their feathery coating.  The wing-like shape your arms once had is gone in a matter of moments, leaving " + player.skinFurScales() + " behind.");
+						message += "  Glancing down in irritation, you discover that your feathery arms are shedding their feathery coating."
+						          +"  The wing-like shape your arms once had is gone in a matter of moments, leaving [skinfurscales] behind.";
 						break;
 
 					case ARM_TYPE_SPIDER:
-						outputText(" your arms' chitinous covering is flaking away.  The glossy black coating is soon gone, leaving " + player.skinFurScales() + " behind.");
+						message += "  Glancing down in irritation, you discover that your arms' chitinous covering is flaking away."
+						          +"  The glossy black coating is soon gone, leaving [skinfurscales] behind.";
 						break;
 
 					case ARM_TYPE_SALAMANDER:
-						outputText(" your once scaly arms are shedding their scales and that your claws become normal human fingernails again.");
+						message += "  Glancing down in irritation, you discover that your once scaly arms are shedding their scales and that"
+						          +" your claws become normal human fingernails again.";
 						break;
 
 					case ARM_TYPE_PREDATOR:
 						switch (player.skinType) {
 							case SKIN_TYPE_GOO:
 								if (player.clawType != CLAW_TYPE_NORMAL)
-									outputText("\n\nYour gooey claws melt into your fingers. Well, who cares, gooey claws aren't very useful in combat to begin with.");
-									//Gooey claws? Really?!? I'll take a look at goo TF later ...
+									message += "\n\nYour gooey claws melt into your fingers."
+									          +" Well, who cares, gooey claws aren't very useful in combat to begin with.";
```

Comment:
This might cause a conflict, because I've already removed that comment in Basilisk Eyes Phase 2.

Resolve by using the two lines starting with `message += "\n\n..."` instead